client/src/components/AddNewProfile.js

```diff
@@ -242,11 +247,11 @@ const AddNewProfile = () => {
         <Dropdown
           id="nationality"
           name="nationality"
-          value={profileData.nationalities}
+          value={profileData.nationality}
```

Comment:
```suggestion
          value={profileData.id}
```

Reply:
You can ignore this as my suggestion is wrong! I got confused—it should stay as `value={profileData.nationality}`